libbpf-tools/profile.c

```diff
@@ -250,13 +252,65 @@ static int cmp_counts(const void *dx, const void *dy)
 	return x > y ? -1 : !(x == y);
 }
 
+static bool batch_map_ops = true; /* hope for the best */
```

Comment:
I do not understand this code.
You want to detect if `bpf_map_lookup_batch()` is available, right?

Reply:
Oh sorry I did not see you set this to false later, this totally makes sense then!